README.md

```diff
@@ -35,11 +35,11 @@ Then, configure your terminal with the desired font, and restart it.
 
 One liner for OS X:
 
-    git clone https://github.com/arialdomartini/oh-my-git.git ~/.oh-my-git && echo source ~/.oh-my-git/prompt.sh >> ~/.profile
+    git clone --depth=1 https://github.com/arialdomartini/oh-my-git.git ~/.oh-my-git && echo source ~/.oh-my-git/prompt.sh >> ~/.profile
```

Comment:
That's smart!
It saves some Kbytes.
```
$ git clone --depth=1 https://github.com/arialdomartini/oh-my-git.git depth-1
$ git clone  https://github.com/arialdomartini/oh-my-git.git depth-all
$ du -sh ./depth1 ./depth-all
156K	./depth1
284K	./depth-all
```

 I'm cherry-picking this commit in a separate PR and merging it to the upstream repo. Then I will go on with the rest of the commits. Thank you, @wwna!